### DISCUSSION TITLE
Showing the registered histology image on the 3D brain mesh model

### DISCUSSION THREAD

**Discussion by David-Omer** [JingyiGF/HERBS#54] [Q&A]
Is it possible to show multiple registered histology slides on the 3D brain mesh?

**Comment by JingyiGF** [comment]
Hmmm, this is interesting question. I remember I tried once for a single perfect cut histological slice. It was ok rendered in 3D. But for matching to rotated atlas slice, the rendering in 3D for histological slice is not so nice. What I am saying is that in 3D meshes/vectors atlases are visualized. But for histological images, they are pixels. It is ok to visualize them if they aligned perfectly vertically or horizontally. If not, they look strange…So I just ignore this functionality. I never tried multiple before. Anyway, I have no time to go into this before Christmas. So if you really need it, you have to convince the group/other users that this is a very useful and important functionality. Then I will  make an effort on it. Best,JingyiSent from my iPhone On 14 Dec 2023, at 16:18, OmerLab ***@***.***> wrote:﻿
Is it possible to show multiple registered histology slides on the 3D brain mesh?
